payments-core/src/test/java/com/stripe/android/view/PaymentUtilsTest.kt

```diff
@@ -68,7 +68,7 @@ class PaymentUtilsTest {
         Locale.setDefault(Locale("ar", "JO")) // Jordan
         val jordanianDinar = Currency.getInstance("JOD")
         assertThat(PaymentUtils.formatPriceString(100123.0, jordanianDinar))
-            .isEqualTo("د.أ.\u200F ١٠٠٫١٢٣")
+            .isEqualTo("\u200F١٠٠٫١٢٣ د.أ.\u200F")
```

Comment:
I have no idea if this change is what we want or not. But CI failed without it.

Reply:
For what it's worth, Stripe doesn't support Jordan as a country, or JOD as a currency.